synapse/replication/tcp/protocol.py

```diff
@@ -83,12 +83,16 @@
     "synapse_replication_tcp_protocol_close_reason", "", ["reason_type"]
 )
 
-tcp_inbound_commands = Counter(
-    "synapse_replication_tcp_protocol_inbound_commands", "", ["command", "name"],
+tcp_inbound_commands_counter = Counter(
+    "synapse_replication_tcp_protocol_inbound_commands",
+    "Number of commands received from replication",
```

Comment:
```suggestion
    "Number of commands received over replication, by command and connection ID",
```

Reply:
They're not connection IDs, they're what they're connected to, will change